annyang.js

```diff
@@ -82,6 +82,12 @@
     return recognition !== undefined;
   };
 
+  // method for logging in developer console when debug mode is on
+  var logMessage = function(text,logStyle){
```

Comment:
Please add a space between the two arguments and between the closing parentheses and opening curly:
````
function(text, logStyle) {
````

I know this seems like nitpicking, but when contributing to an open source project, make sure to follow the established conventions. That doesn't mean the conventions I chose are better than others, they just need to be consistent :)